### PR TITLE
Add Animation::GetTrack(unsigned)

### PR DESCRIPTION
**Pull request by Florastamine**
This variant of Animation::GetTrack() is listed in the documentation but I couldn't see it implemented anywhere in the source code.

### CODE REVIEW

File: Source/Urho3D/Graphics/Animation.cpp [hunk 1]
```diff
@@ -376,8 +376,25 @@ SharedPtr<Animation> Animation::Clone(const String& cloneName) const
     ret->SetMemoryUse(GetMemoryUse());
     
     return ret;
+} 
+
+AnimationTrack* Animation::GetTrack(unsigned index) 
+{
+    index = (index < 0) || (index > tracks_.Size() - 1) ? 0 : index;
```
Comment (eugeneko) [inline Source/Urho3D/Graphics/Animation.cpp]: Ewww... Let's just return 0, as everywhere.

**Comment by cadaver** [comment]
AngelScript binding for the new overload missing? It's OK to be just a GetTrack function, since the property access happens with a string. 

And yes, agree with @eugeneko, if index is illegal, just early-out to returning a null pointer.

**Comment by Florastamine** [comment]
Done. Apologies for the obvious mistakes.

**Comment by Mike3D** [comment]
Wasn't this already exposed by 'tracks' array ?

**Comment by cadaver** [comment]
Thanks, will wait for the CI and merge then.

It's OK to have a convenience overload that accesses by index directly.

**Pull request merged**